.helm/templates/crd-sql-server.yaml

```diff
@@ -97,7 +97,7 @@ spec:
                   type: integer
                   description: How long to wait before polling for next result set. Can be from 1 to 60 seconds.
```

Comment:
Should the description be modified accordingly?

Reply:
Thanks, will fix that :)